packages/terra-functional-testing/src/reporters/wdio-accessibility-reporter.js

```diff
@@ -0,0 +1,123 @@
+const chalk = require('chalk');
+const WDIOReporter = require('@wdio/reporter').default;
+
+class AccessibilityReporter extends WDIOReporter {
+  /**
+   * Indents a string.
+   * @param {string} string - The string to indent.
+   * @param {number} indent - The number of spaces to indent the string.
+   */
+  static indent(string, indent) {
+    if (indent > 0) {
+      return string.replace(/^(?!\s*$)/gm, Array(indent + 1).join(' '));
+    }
+
+    return string;
+  }
+
+  constructor(options) {
+    super(options);
+
+    this.currentTest = null;
+    this.accessibilityResults = {};
+
+    process.on('terra:report:accessibility', this.onReportAccessibility.bind(this));
+  }
+
+  /**
+   * Hook invoked at the start of the test.
+   * @param {TestStats} test - The current test object.
+   */
+  onTestStart(test) {
+    this.currentTest = test.uid;
+  }
+
+  /**
+   * Hook invoked when accessibility results are reported.
+   * @param {Object} results - Accessibility results.
+   * @param {array} results.incomplete - Incomplete accessibility results.
+   */
+  onReportAccessibility(results) {
+    this.accessibilityResults[this.currentTest] = results;
```

Comment:
Is it possible to know the suite info here instead of recursively building it out at the end?

Reply:
Probably, but I personally think traveling the tree is actually easier than tracking additional test data manually. A suite can be nested infinitely inside of other suites